/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/OpenSSL/crypto.py:12: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.
  from cryptography import x509
[2020-12-05 13:16:32 +0530] [23805] [INFO] Starting gunicorn 19.10.0
[2020-12-05 13:16:32 +0530] [23805] [INFO] Listening at: http://127.0.0.1:8000 (23805)
[2020-12-05 13:16:32 +0530] [23805] [INFO] Using worker: sync
[2020-12-05 13:16:32 +0530] [23869] [INFO] Booting worker with pid: 23869
[2020-12-05 13:21:44 +0530] [23805] [CRITICAL] WORKER TIMEOUT (pid:23869)
[2020-12-05 13:21:44 +0530] [23869] [ERROR] Error handling request /api/method/frappe.desk.page.setup_wizard.setup_wizard.setup_complete
Traceback (most recent call last):
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/app.py", line 64, in application
    response = frappe.api.handle()
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/api.py", line 59, in handle
    return frappe.handler.handle()
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/handler.py", line 64, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/__init__.py", line 1064, in call
    return fn(*args, **newargs)
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/desk/page/setup_wizard/setup_wizard.py", line 72, in setup_complete
    task.get('fn')(task.get('args'))
  File "/home/erpnext2/frappe-bench/apps/erpnext/erpnext/setup/setup_wizard/setup_wizard.py", line 95, in setup_company
    fixtures.install_company(args)
  File "/home/erpnext2/frappe-bench/apps/erpnext/erpnext/setup/setup_wizard/operations/install_fixtures.py", line 396, in install_company
    make_records(records)
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/desk/page/setup_wizard/setup_wizard.py", line 392, in make_records
    doc.insert(ignore_permissions=True)
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/model/document.py", line 260, in insert
    self.run_post_save_methods()
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/model/document.py", line 927, in run_post_save_methods
    self.run_method("on_update")
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/model/document.py", line 797, in run_method
    out = Document.hook(fn)(self, *args, **kwargs)
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/model/document.py", line 1073, in composer
    return composed(self, method, *args, **kwargs)
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/model/document.py", line 1056, in runner
    add_to_return_value(self, fn(self, *args, **kwargs))
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/model/document.py", line 791, in <lambda>
    fn = lambda self, *args, **kwargs: getattr(self, method)(*args, **kwargs)
  File "/home/erpnext2/frappe-bench/apps/erpnext/erpnext/setup/doctype/company/company.py", line 107, in on_update
    install_country_fixtures(self.name)
  File "/home/erpnext2/frappe-bench/apps/erpnext/erpnext/setup/doctype/company/company.py", line 419, in install_country_fixtures
    frappe.get_attr(module_name)(company_doc, False)
  File "/home/erpnext2/frappe-bench/apps/erpnext/erpnext/regional/india/setup.py", line 14, in setup
    setup_company_independent_fixtures()
  File "/home/erpnext2/frappe-bench/apps/erpnext/erpnext/regional/india/setup.py", line 21, in setup_company_independent_fixtures
    make_custom_fields()
  File "/home/erpnext2/frappe-bench/apps/erpnext/erpnext/regional/india/setup.py", line 509, in make_custom_fields
    create_custom_fields(custom_fields, update=update)
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/custom/doctype/custom_field/custom_field.py", line 143, in create_custom_fields
    create_custom_field(doctype, df, ignore_validate=ignore_validate)
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/custom/doctype/custom_field/custom_field.py", line 127, in create_custom_field
    custom_field.insert()
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/model/document.py", line 260, in insert
    self.run_post_save_methods()
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/model/document.py", line 927, in run_post_save_methods
    self.run_method("on_update")
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/model/document.py", line 797, in run_method
    out = Document.hook(fn)(self, *args, **kwargs)
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/model/document.py", line 1073, in composer
    return composed(self, method, *args, **kwargs)
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/model/document.py", line 1056, in runner
    add_to_return_value(self, fn(self, *args, **kwargs))
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/model/document.py", line 791, in <lambda>
    fn = lambda self, *args, **kwargs: getattr(self, method)(*args, **kwargs)
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/custom/doctype/custom_field/custom_field.py", line 61, in on_update
    validate_fields_for_doctype(self.dt)
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/core/doctype/doctype/doctype.py", line 672, in validate_fields_for_doctype
    doc = frappe.get_doc("DocType", doctype)
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/__init__.py", line 762, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/model/document.py", line 71, in get_doc
    return controller(*args, **kwargs)
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/model/document.py", line 106, in __init__
    self.load_from_db()
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/model/document.py", line 162, in load_from_db
    "*", as_dict=True, order_by="idx asc")
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/database/database.py", line 445, in get_values
    out = self._get_values_from_table(fields, filters, doctype, as_dict, debug, order_by, update)
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/database/database.py", line 592, in _get_values_from_table
    as_dict=as_dict, debug=debug, update=update)
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/database/database.py", line 156, in sql
    self._cursor.execute(query, values)
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/pymysql/cursors.py", line 170, in execute
    result = self._query(query)
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/pymysql/cursors.py", line 328, in _query
    conn.query(q)
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/pymysql/connections.py", line 517, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/pymysql/connections.py", line 732, in _read_query_result
    result.read()
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/pymysql/connections.py", line 1082, in read
    self._read_result_packet(first_packet)
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/pymysql/connections.py", line 1152, in _read_result_packet
    self._read_rowdata_packet()
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/pymysql/connections.py", line 1190, in _read_rowdata_packet
    rows.append(self._read_row_from_packet(packet))
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/pymysql/connections.py", line 1209, in _read_row_from_packet
    data = converter(data)
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/gunicorn/workers/base.py", line 202, in handle_abort
    sys.exit(1)
SystemExit: 1

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/gunicorn/workers/sync.py", line 176, in handle_request
    respiter = self.wsgi(environ, resp.start_response)
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/werkzeug/local.py", line 231, in application
    return ClosingIterator(app(environ, start_response), self.cleanup)
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/werkzeug/wrappers/base_request.py", line 237, in application
    resp = f(*args[:-2] + (request,))
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/app.py", line 92, in application
    frappe.db.rollback()
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/database/database.py", line 753, in rollback
    self.sql("rollback")
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/database/database.py", line 171, in sql
    self._cursor.execute(query)
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/pymysql/cursors.py", line 170, in execute
    result = self._query(query)
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/pymysql/cursors.py", line 328, in _query
    conn.query(q)
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/pymysql/connections.py", line 517, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/pymysql/connections.py", line 732, in _read_query_result
    result.read()
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/pymysql/connections.py", line 1075, in read
    first_packet = self.connection._read_packet()
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/pymysql/connections.py", line 671, in _read_packet
    % (packet_number, self._next_seq_id))
pymysql.err.InternalError: Packet sequence number wrong - got 218 expected 1
[2020-12-05 13:21:44 +0530] [23869] [INFO] Worker exiting (pid: 23869)
[2020-12-05 13:21:45 +0530] [24038] [INFO] Booting worker with pid: 24038
[2020-12-05 13:25:12 +0530] [23805] [INFO] Handling signal: term
[2020-12-05 13:25:12 +0530] [24038] [INFO] Worker exiting (pid: 24038)
[2020-12-05 13:25:13 +0530] [23805] [INFO] Shutting down: Master
/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/OpenSSL/crypto.py:12: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.
  from cryptography import x509
[2020-12-05 13:25:30 +0530] [24209] [INFO] Starting gunicorn 19.10.0
[2020-12-05 13:25:30 +0530] [24209] [INFO] Listening at: http://127.0.0.1:8000 (24209)
[2020-12-05 13:25:30 +0530] [24209] [INFO] Using worker: sync
[2020-12-05 13:25:30 +0530] [24246] [INFO] Booting worker with pid: 24246
[2020-12-05 13:28:34 +0530] [24209] [INFO] Handling signal: term
[2020-12-05 13:28:34 +0530] [24246] [INFO] Worker exiting (pid: 24246)
[2020-12-05 13:28:34 +0530] [24209] [INFO] Shutting down: Master
/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/OpenSSL/crypto.py:12: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.
  from cryptography import x509
[2020-12-05 13:28:47 +0530] [24402] [INFO] Starting gunicorn 19.10.0
[2020-12-05 13:28:47 +0530] [24402] [INFO] Listening at: http://127.0.0.1:8000 (24402)
[2020-12-05 13:28:47 +0530] [24402] [INFO] Using worker: sync
[2020-12-05 13:28:47 +0530] [24430] [INFO] Booting worker with pid: 24430
[2020-12-05 13:34:25 +0530] [24402] [INFO] Handling signal: term
[2020-12-05 13:34:25 +0530] [24430] [INFO] Worker exiting (pid: 24430)
[2020-12-05 13:34:25 +0530] [24402] [INFO] Shutting down: Master
/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/OpenSSL/crypto.py:12: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.
  from cryptography import x509
[2020-12-05 13:34:38 +0530] [24627] [INFO] Starting gunicorn 19.10.0
[2020-12-05 13:34:38 +0530] [24627] [INFO] Listening at: http://127.0.0.1:8000 (24627)
[2020-12-05 13:34:38 +0530] [24627] [INFO] Using worker: sync
[2020-12-05 13:34:38 +0530] [24658] [INFO] Booting worker with pid: 24658
[2020-12-05 13:40:22 +0530] [24627] [INFO] Handling signal: term
[2020-12-05 13:40:22 +0530] [24658] [INFO] Worker exiting (pid: 24658)
[2020-12-05 13:40:23 +0530] [24627] [INFO] Shutting down: Master
/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/OpenSSL/crypto.py:12: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.
  from cryptography import x509
[2020-12-05 13:40:36 +0530] [24837] [INFO] Starting gunicorn 19.10.0
[2020-12-05 13:40:36 +0530] [24837] [INFO] Listening at: http://127.0.0.1:8000 (24837)
[2020-12-05 13:40:36 +0530] [24837] [INFO] Using worker: sync
[2020-12-05 13:40:36 +0530] [24868] [INFO] Booting worker with pid: 24868
[2020-12-05 13:54:36 +0530] [24837] [INFO] Handling signal: term
[2020-12-05 13:54:36 +0530] [24868] [INFO] Worker exiting (pid: 24868)
[2020-12-05 13:54:37 +0530] [24837] [INFO] Shutting down: Master
/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/OpenSSL/crypto.py:12: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.
  from cryptography import x509
[2020-12-05 13:54:55 +0530] [25217] [INFO] Starting gunicorn 19.10.0
[2020-12-05 13:54:55 +0530] [25217] [INFO] Listening at: http://127.0.0.1:8000 (25217)
[2020-12-05 13:54:55 +0530] [25217] [INFO] Using worker: sync
[2020-12-05 13:54:55 +0530] [25268] [INFO] Booting worker with pid: 25268
[2020-12-05 13:58:17 +0530] [25217] [CRITICAL] WORKER TIMEOUT (pid:25268)
[2020-12-05 13:58:17 +0530] [25268] [INFO] Worker exiting (pid: 25268)
[2020-12-05 13:58:18 +0530] [25402] [INFO] Booting worker with pid: 25402
[2020-12-05 14:00:05 +0530] [25217] [INFO] Handling signal: term
[2020-12-05 14:00:05 +0530] [25402] [INFO] Worker exiting (pid: 25402)
[2020-12-05 14:00:05 +0530] [25217] [INFO] Shutting down: Master
/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/OpenSSL/crypto.py:12: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.
  from cryptography import x509
[2020-12-05 14:00:12 +0530] [25477] [INFO] Starting gunicorn 19.10.0
[2020-12-05 14:00:12 +0530] [25477] [INFO] Listening at: http://127.0.0.1:8000 (25477)
[2020-12-05 14:00:12 +0530] [25477] [INFO] Using worker: sync
[2020-12-05 14:00:12 +0530] [25510] [INFO] Booting worker with pid: 25510
[2020-12-05 14:06:10 +0530] [25510] [ERROR] Error handling request /api/method/frappe.core.page.background_jobs.background_jobs.get_scheduler_status
Traceback (most recent call last):
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/gunicorn/workers/sync.py", line 176, in handle_request
    respiter = self.wsgi(environ, resp.start_response)
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/werkzeug/local.py", line 231, in application
    return ClosingIterator(app(environ, start_response), self.cleanup)
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/werkzeug/wrappers/base_request.py", line 237, in application
    resp = f(*args[:-2] + (request,))
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/app.py", line 88, in application
    rollback = after_request(rollback)
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/app.py", line 215, in after_request
    updated_in_db = frappe.local.session_obj.update()
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/sessions.py", line 362, in update
    "name": frappe.session.user
  File "/home/erpnext2/frappe-bench/apps/frappe/frappe/database/database.py", line 156, in sql
    self._cursor.execute(query, values)
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/pymysql/cursors.py", line 170, in execute
    result = self._query(query)
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/pymysql/cursors.py", line 328, in _query
    conn.query(q)
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/pymysql/connections.py", line 517, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/pymysql/connections.py", line 732, in _read_query_result
    result.read()
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/pymysql/connections.py", line 1075, in read
    first_packet = self.connection._read_packet()
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/pymysql/connections.py", line 684, in _read_packet
    packet.check_error()
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/pymysql/protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/pymysql/err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.InternalError: (1205, 'Lock wait timeout exceeded; try restarting transaction')
[2020-12-05 14:27:18 +0530] [25477] [INFO] Handling signal: term
[2020-12-05 14:27:18 +0530] [25510] [INFO] Worker exiting (pid: 25510)
[2020-12-05 14:27:19 +0530] [25477] [INFO] Shutting down: Master
/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/OpenSSL/crypto.py:12: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.
  from cryptography import x509
[2020-12-07 09:10:34 +0530] [1337] [INFO] Starting gunicorn 19.10.0
[2020-12-07 09:10:34 +0530] [1337] [INFO] Listening at: http://127.0.0.1:8000 (1337)
[2020-12-07 09:10:34 +0530] [1337] [INFO] Using worker: sync
[2020-12-07 09:10:35 +0530] [1895] [INFO] Booting worker with pid: 1895
[2020-12-07 13:01:36 +0530] [1337] [INFO] Handling signal: term
[2020-12-07 13:01:36 +0530] [1895] [INFO] Worker exiting (pid: 1895)
[2020-12-07 13:01:38 +0530] [1337] [INFO] Shutting down: Master
/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/OpenSSL/crypto.py:12: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.
  from cryptography import x509
[2020-12-07 22:37:17 +0530] [1429] [INFO] Starting gunicorn 19.10.0
[2020-12-07 22:37:17 +0530] [1429] [INFO] Listening at: http://127.0.0.1:8000 (1429)
[2020-12-07 22:37:17 +0530] [1429] [INFO] Using worker: sync
[2020-12-07 22:37:17 +0530] [2356] [INFO] Booting worker with pid: 2356
/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/OpenSSL/crypto.py:12: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.
  from cryptography import x509
[2020-12-08 09:29:56 +0530] [1440] [INFO] Starting gunicorn 19.10.0
[2020-12-08 09:29:56 +0530] [1440] [INFO] Listening at: http://127.0.0.1:8000 (1440)
[2020-12-08 09:29:56 +0530] [1440] [INFO] Using worker: sync
[2020-12-08 09:29:56 +0530] [2135] [INFO] Booting worker with pid: 2135
/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/OpenSSL/crypto.py:12: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.
  from cryptography import x509
[2020-12-10 11:44:18 +0530] [1363] [INFO] Starting gunicorn 19.10.0
[2020-12-10 11:44:18 +0530] [1363] [INFO] Listening at: http://127.0.0.1:8000 (1363)
[2020-12-10 11:44:18 +0530] [1363] [INFO] Using worker: sync
[2020-12-10 11:44:18 +0530] [1624] [INFO] Booting worker with pid: 1624
/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/OpenSSL/crypto.py:12: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.
  from cryptography import x509
[2020-12-10 23:48:04 +0530] [1375] [INFO] Starting gunicorn 19.10.0
[2020-12-10 23:48:04 +0530] [1375] [INFO] Listening at: http://127.0.0.1:8000 (1375)
[2020-12-10 23:48:04 +0530] [1375] [INFO] Using worker: sync
[2020-12-10 23:48:04 +0530] [1625] [INFO] Booting worker with pid: 1625
[2020-12-11 00:45:50 +0530] [1375] [INFO] Handling signal: term
[2020-12-11 00:45:50 +0530] [1625] [INFO] Worker exiting (pid: 1625)
[2020-12-11 00:45:51 +0530] [1375] [INFO] Shutting down: Master
/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/OpenSSL/crypto.py:12: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.
  from cryptography import x509
[2020-12-11 10:47:56 +0530] [1458] [INFO] Starting gunicorn 19.10.0
[2020-12-11 10:47:56 +0530] [1458] [INFO] Listening at: http://127.0.0.1:8000 (1458)
[2020-12-11 10:47:56 +0530] [1458] [INFO] Using worker: sync
[2020-12-11 10:47:56 +0530] [2006] [INFO] Booting worker with pid: 2006
/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/OpenSSL/crypto.py:12: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.
  from cryptography import x509
[2020-12-12 09:15:23 +0530] [1423] [INFO] Starting gunicorn 19.10.0
[2020-12-12 09:15:23 +0530] [1423] [INFO] Listening at: http://127.0.0.1:8000 (1423)
[2020-12-12 09:15:23 +0530] [1423] [INFO] Using worker: sync
[2020-12-12 09:15:23 +0530] [2419] [INFO] Booting worker with pid: 2419
/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/OpenSSL/crypto.py:12: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.
  from cryptography import x509
[2020-12-14 09:57:33 +0530] [1394] [INFO] Starting gunicorn 19.10.0
[2020-12-14 09:57:33 +0530] [1394] [INFO] Listening at: http://127.0.0.1:8000 (1394)
[2020-12-14 09:57:33 +0530] [1394] [INFO] Using worker: sync
[2020-12-14 09:57:33 +0530] [2328] [INFO] Booting worker with pid: 2328
/home/erpnext2/frappe-bench/env/lib/python3.5/site-packages/OpenSSL/crypto.py:12: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.
  from cryptography import x509
[2020-12-16 09:35:44 +0530] [1447] [INFO] Starting gunicorn 19.10.0
[2020-12-16 09:35:44 +0530] [1447] [INFO] Listening at: http://127.0.0.1:8000 (1447)
[2020-12-16 09:35:44 +0530] [1447] [INFO] Using worker: sync
[2020-12-16 09:35:45 +0530] [2840] [INFO] Booting worker with pid: 2840
